pydantic/class_validators.py

```diff
@@ -12,6 +12,8 @@
 
 
 class Validator:
+    # Applies only to RootValidators
+    SKIP_ON_FAILURE_KEY = '__validator_skip_on_failure__'
```

Comment:
```suggestion
    SKIP_ON_FAILURE_KEY = '__validator_skip_on_failure__'  # Applies only to RootValidators
```

I find it slightly less confusing to have the comment on the same line.